_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+### Update your Software
+* Especially for Zoom, you will want to be using the most recent version and the same version as everyone else. Remind your learners to update as well!
+## 2. Workshop Mechanics
+* **Assign a "Zoom DJ"** to assume the "host" role in the meeting. A Zoom DJ will manage breakouts, waiting rooms, muting, and troubleshooting as needed. This role is typically played by a Supporting Instructor; if/when they choose to teach, plan to hand off this role to another Instructor.
+* **Customize profile names** to clearly indicate instructors and helpers. For example, a helper named Ada might change their profile to read "Helper: Ada (she/her)"
+* **Use Zoom features as you would use sticky notes**, and remember to call upon learners to use them regularly. 
+    * On Zoom, learners can choose "reactions" that include a green '✓' and a red 'x' icon. Unlike other reactions, these persist until cleared.
+* **Use breakout rooms** for socialization and activities whenever possible. These do take time and that may mean cutting content. The benefits to creating a supportive learning environment make this cost worthwhile.
+    * Using a shared timer can help, especially as learners get used to using it.
+* **Ask learners to keep the chat focused.** You may wish to provide them with an alternate venue to share social chat.
+* **Use a 3-step help response system**.
+    * Step 1: Learners message a helper, who helps via **direct messaging** in the conference chat if they can. 
+    * Step 2: Helpers communicate about questions they have received via **back-channel conversations** (even if resolved). This will elevate common questions that merit attention of the teaching Instructor and will allow others to advise if the problem has not been solved.
+    * Step 3: For difficult problems, invite the learner to **share their screen** for live support. Depending on the workshop size, frequency of the problem, etc., this may be done in the main room as a pause in the workshop or in a breakout room with a helper.
+        * If you create breakout rooms for activities, it helps to create extra rooms for this purpose at that time so that activity rooms can be preserved.
+        * Keep in mind that learners in breakout rooms will miss ongoing instruction in the main room, so helpers should also help learners to catch up before returning.
+* **Divert distractions and disruptions to breaks** wherever possible. Advanced questions and other difficult conversations can be resolved using breakout rooms during breaks. If you have plenty of breaks available, missing one to problem solve will be less strenuous.
+
+## 3. Assessment
+* **Formative assessment is more important than ever**. When teaching online, you lose the ability to visually "read the room," especially if learners' cameras are turned off. That's ok, because frequent formative assessment will provide the feedback you need. 
```

Comment:
Consider replacing 'read the room' as idioms don't always translate globally

Reply:
Break out rooms caused issues in the South African context. We asked students to share their screens rather and we could all trouble shoot together.